signing_service/signing_service/signing_service.py

```diff
@@ -384,6 +419,34 @@ def _get_authentication_challenge_signature(self, authentication_challenge: byte
             authentication_challenge,
         )
 
+    @staticmethod
+    def _get_daily_transaction_threshold_file_path() -> PosixPath:
+        thresholds_directory = (Path.cwd()).joinpath('daily_tresholds')  # pylint: disable=no-member
+        thresholds_directory.mkdir(exist_ok=True)  # pylint: disable=no-member
+        daily_file = datetime.datetime.now().strftime('%Y-%m-%d')
+        thresholds_directory.joinpath(daily_file).touch(exist_ok=True)  # pylint: disable=no-member
+        thresholds_directory = thresholds_directory.joinpath(daily_file)
```

Comment:
you are getting a `tresholds_file_path` - why overriding `tresholds_directory`?

Reply:
No reason, I'll fix it.